<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QDialog" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="username_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>128</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="password_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>168</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fname_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>208</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change first name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>288</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>368</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change city</string>
   </property>
  </widget>
  <widget class="QPushButton" name="address_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>328</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Change info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new pasword:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new first name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fname_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new city:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="city_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="lname_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>248</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change last name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new last name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lname_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
